docs/reference/api_examples.yml

```diff
@@ -249,6 +249,11 @@
 
   doc: |
     Get a list of all datasets in a deployment.
+    This includes manifest (attached) and non-manifest (unattached) datasets.
+    Some examples of non-manifest datasets are datasets that are being created or
```

Comment:
This is still confusing, I think. How about "For example, a block device volume that has been created but is not attached to any machine." or something along those lines?